build.gradle

```diff
@@ -53,8 +53,8 @@ allprojects {
             downloadSources = true
         }
     }
-    sourceCompatibility = '1.8'
-    targetCompatibility = '1.8'
+    sourceCompatibility = "11"
```

Comment:
Library source and target compatibility should remain 1.8 for radar-auth, since it’s used as a library potentially in Java 8 projects. For the root project, Java 11 is fine, as well as for managementportal-client: all kotlin projects can be expected to be migrated to Java 11 by now.

Reply:
That makes sense. Didn't know radar-auth is a shared lib. Change reverted.

In that case, should radar-auth be better extracted into its own codebase instead of being inside this mono-repo? Of cause, in a separate issue.

Reply:
The authorization library is closely tied to managementportal, so we'd like to keep them together. An improvement to the repository structure could be to make the managementportal root module a submodule as well, to clarify the relationship between the modules.

Reply:
Indeed, perhaps take out sourceCompatibility out of the `allprojects` block and add it manually to each project with the appropriate version:
radar-auth: 1.8
radar-auth/deprecated-auth0: 1.8
root: 11
managemeåtportal-client: 11
oauth-client: 1.8

Reply:
Yes, let's do that.